### ISSUE TITLE
Admins of a group should be able to censor a message

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/cyclescape#1008]
Apparently, group admins aren't able to censor a message.

This should be fixed.

**Comment by nikolai-b** [comment]
I don't think we have group admins, just committee members.  Committee members can censor messages in their group so I guess this is because the thread is in their area but not marked as a group thread.

**Comment by mvl22** [comment]
Sorry, I did mean committee members indeed.

I’ve had enough feedback on this from committee members of a group whose messages are almost all private to be fairly sure there’s a bug.

**Comment by nikolai-b** [comment]
Ah, sounds like a bug. Can you give a specific user and message ID?

**Comment by mvl22** [comment]
@roxannedebeaux Could you give an example thread ID for a non-public thread, where you aren't able to hide a message?

You should be seeing a cog like this, in the top-right of a message:

![Screenshot 2022-01-02 at 23 26 44](https://user-images.githubusercontent.com/361423/147892194-6e79d814-e1ec-47d4-b608-7f5f39e66503.png)
